### ISSUE TITLE
Deploy the NLP connector to SaaS users for testing

### ISSUE THREAD

**Issue by nino-filigran** [LuatixHQ/opencti#5652]
Goal of the task is to make the solution avaialble to SaaS users.

**Comment by ghost** [comment]
@Jipegien I think the best and easiest way for you to leverage NER (Named Entity Recognition) to extract entities and relations out of unstructured texts is to use AI integration (OpenAI or your custom trained Mistral model).
I already use such an approach before ingestion.
Before I used this approach I tried it with well-known python frameworks with medium good results. After moving to NER with an AI model (First tried OpenAI, then Mistral; stayed at mistral because OpenAI gets to expensive if you run this on random web scraped content) the results were pretty good.
Hope it helps a little.

**Comment by Jipegien** [comment]
Yes we are considering this possibility, but in a context of multiple users, it comes rapidly with huge costs and latencies. Results are often not so consistent over time. Thanks :)
